### DISCUSSION TITLE
Support for donations / sponsorship via Github Sponsors

### DISCUSSION THREAD

**Discussion by Kami** [mastodon/mastodon#21694] [General]
First thanks to everyone who has contributed to this great project.

I would love to sponsor the project, but I prefer to use Github Sponsors for that (it makes it easier for me to manage for various projects in a single location).

Do you have any plans to support sponsorships / donations via Github sponsors?

Thanks.

**Comment by mjankowski** [comment]
If still curious/relevant, yes, sponsorships are enabled: https://github.com/sponsors/mastodon
